src/main/java/hudson/plugins/ec2/EC2Cloud.java

```diff
@@ -762,7 +762,7 @@ protected FormValidation doTestConnection(URL ec2endpoint, boolean useInstancePr
                     EC2PrivateKey pk = new EC2PrivateKey(privateKey);
                     if (pk.find(ec2) == null)
                         return FormValidation
-                                .error("The EC2 key pair private key isn't registered to this EC2 region (fingerprint is "
+                                .warning("The EC2 key pair private key isn't registered to this EC2 region (fingerprint is "
                                         + pk.getFingerprint() + ")");
```

Comment:
I would give a hint here about the reason you would not specify the private key since it's no longer required.